### ISSUE TITLE
Proper H1 structure for screens

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/area-model-common#149]
The a11y design notes we'll want H1s for the screens (including the home screen). I'm not sure if the design is finalized, but this should be included in this sim before the 1.1 release.

@zepumph, do you know the current status of this, or if there's an issue I can link here?

**Comment by zepumph** [comment]
I created https://github.com/phetsims/a11y-research/issues/104, hopefully that will help.

**Comment by terracoda** [comment]
@jonathanolson, I have created a design pattern in [joist/doc/HomeScreen.md](https://github.com/phetsims/joist/blob/master/doc/HomeScreen.md) and updated the HTML example for [Build an Atom](https://github.com/phetsims/a11y-research/blob/master/html-sketches/sim-screens/baa-screens/index.html) to follow that pattern.

Please discuss with @amanda-phet.

I marked everything ready-for-review, but there are several issues related to this topic of the Home Screen.

**Comment by amanda-phet** [comment]
I updated the H1s in the descriptions design doc.

**Comment by jonathanolson** [comment]
@jessegreenberg or @zepumph, how would I implement this?

**Comment by zepumph** [comment]
I want to talk to @jessegreenberg about this, I think we may go with a general solution on the ScreenView so you won't have to do anything.

**Comment by terracoda** [comment]
@zepumph and @jessegreenberg, there was positive feedback from designers in phetsims/a11y-research/issues/112

It the approach described in the HomeScreen.md file.

**Comment by zepumph** [comment]
This will take a bit more work than I can do today, I'll work on it more tomorrow.

**Comment by zepumph** [comment]
I updated the way that the headers are created for multiscreen sims. I also simplified the implementation so that we could clean up code in single screen simulations. Sorry that this sim specific issue ended up being joist based and cross sim. 

From here we will continue implementation of HomeScreen.md in https://github.com/phetsims/joist/issues/496 so please general comments over there. 

@jonathanolson please let me know if you need anything else for area model specifically.

**Comment by jonathanolson** [comment]
Nothing really specific to this sim (will close). Added to https://github.com/phetsims/area-model-common/issues/154.
